storage/src/adb/any.rs

```diff
@@ -735,34 +718,35 @@ mod test {
     pub fn test_any_db_empty() {
         let executor = deterministic::Runner::default();
         executor.start(|context| async move {
+            let mut db = open_db(context.clone()).await;
             let mut hasher = Sha256::new();
-            let mut db = open_db(context.clone(), &mut hasher).await;
+            let mut hasher = Basic::new(&mut hasher);
             assert_eq!(db.op_count(), 0);
             assert_eq!(db.oldest_retained_loc(), None);
             assert!(matches!(db.prune_inactive().await, Ok(())));
             assert_eq!(db.root(&mut hasher), MemMmr::default().root(&mut hasher));
 
             // Make sure closing/reopening gets us back to the same state, even after adding an uncommitted op.
-            let d1 = <Sha256 as CHasher>::Digest::decode(vec![1u8; 32].as_ref()).unwrap();
-            let d2 = <Sha256 as CHasher>::Digest::decode(vec![2u8; 32].as_ref()).unwrap();
+            let d1 = <Sha256 as CHasher>::Digest::decode(vec![1u8; SHA256_SIZE].as_ref()).unwrap();
```

Comment:
IMO we should create a helper on `Sha256` that replicates the same `byte` for the entire duration (like I did with `random`):

https://github.com/commonwarexyz/monorepo/blob/40b0db0fc5aa7e725d03a6f510c6ea4f76f33328/cryptography/src/sha256/mod.rs#L84-L89

Reply:
done!  named the method 'fill' for lack of better ideas.